src/vaadin-avatar.js

```diff
@@ -186,6 +190,17 @@ class AvatarElement extends ElementMixin(ThemableMixin(PolymerElement)) {
         observer: '__colorIndexChanged'
       },
 
+      /**
+       * Clickable changes the accessibility property `role` on the element from `img` to `button`.
+       * This indicates to screenreaders that the user can interact with the element.
+       * It also changes the cursor to pointer to indicate that the element can be clicked.
+       * Set `clickable` to true if you attach a click listener to the element, to indicate that the user can interact it
+       */
+      clickable: {
```

Comment:
The linked issue was only about fixing the default role of `<vaadin-avatar>`, not about adding new features / API. It would be great if you could adapt this PR to only contain the fix.

If you need a feature where the elements behaviour can be toggled between img / button, then please open a new issue in the web-components repo in which you describe your use-case. That would allow us (DS team) to plan and discuss a possible API for what you are trying to accomplish.

Personally I'm not a fan of the `clickable` API. It might make more sense to allow setting the existing `role` attribute and change styles based on that.